src/main/kotlin/no/nav/tpts/mottak/db/DataSource.kt

```diff
@@ -3,18 +3,30 @@ package no.nav.tpts.mottak.db
 import com.zaxxer.hikari.HikariDataSource
 import kotliquery.Connection
 import kotliquery.Session
-import org.postgresql.ds.PGSimpleDataSource
+import mu.KotlinLogging
 
 const val MAX_POOLS = 10
 
+private val LOG = KotlinLogging.logger {}
+
 object DataSource {
     private fun init(): HikariDataSource {
+        LOG.info {
+            "Kobler til Postgres med URL ${
+                System.getenv("NAIS_DATABASE_TPTS_TILTAKSPENGER_MOTTAK_DB_URL")
```

Comment:
Kan lage en `getEnvOrProperty` funksjon så blir det litt lettere å lese / mindre sjanse for å copy-paste feil env-variabelnavn